### PR TITLE
add SliceTracker

### PR DESCRIPTION
**Pull request by fedorov**
This is a workflow module to support MRI-guided prostate biopsy

https://github.com/SlicerProstate/SliceTracker

http://wiki.slicer.org/slicerWiki/index.php/Documentation/Nightly/Extensions/SliceTracker

### CODE REVIEW

File: SliceTracker.s4ext [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#
+# First token of each non-comment line is the keyword and the rest of the line
+# (including spaces) is the value.
+# - the value can be blank
+#
+
+# This is source code manager (i.e. svn)
+scm git
+scmurl https://github.com/SlicerProstate/SliceTracker.git
+scmrevision master
```
Comment (fedorov) [inline SliceTracker.s4ext]: @che85 FYI this points to master, which means any changes to the master will propagate automatically to the packaged extension, and we should be careful what is merged into master (once the extension stabilizes)

Comment (che85) [inline SliceTracker.s4ext]: Alright

**Comment by fedorov** [comment]
@PeterBehringer - FYI - we finally added this as a Slicer extension!

**Comment by PeterBehringer** [comment]
wow, this is a great milestone! thank you for mentioning me as contributor and all the hard work you spent!

**Comment by fedorov** [comment]
What was really amazing that the extension built without errors on the mac dashboard right away, and the next day (yesterday) we were able to use the packaged version of the extension during a biopsy procedure at [AMIGO](http://www.brighamandwomens.org/research/amigo/default.aspx)! I could not believe this was happening! :)

Here are @che85, Kemal Tuncali and @fedorov on that remarkable day!
img_20160714_130644158

Not that we didn't find new issues, there is definitely work to be done https://github.com/SlicerProstate/SliceTracker/milestone/1, but IMO this was a great success!

**Comment by jcfr** [comment]
Congrats and thanks for sharing the good news :+1:

**Comment by pieper** [comment]
Excellent work!

**Comment by PeterBehringer** [comment]
So great to see this! very well done guys.

**Pull request merged**